packages/addons/addon-depends/snapcast-depends/snapcast/package.mk

```diff
@@ -0,0 +1,45 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2018-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="snapcast"
+PKG_VERSION="fcf5997"
+PKG_SHA256="9ef3d42b8e3aaa68eb8742b0ce43e13e94df3f5605e6b8c651b4daffaaa271b6"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv3"
+PKG_SITE="https://github.com/badaix/snapcast"
+PKG_URL="https://github.com/badaix/$PKG_NAME/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain aixlog alsa-lib asio jsonrpcpp avahi flac libvorbis popl"
+PKG_SECTION="tools"
+PKG_SHORTDESC="snapcast"
+PKG_LONGDESC="Synchronous multi-room audio player"
+
+PKG_TOOLCHAIN="make"
+
+pre_configure_target() {
```

Comment:
shouldn't it be ?
```
pre_configure_target() {
  rm -rf externals
  mkdir externals
    cp -r $(get_build_dir aixlog) externals/aixlog
    cp -r $(get_build_dir asio) externals/asio
    cp -r $(get_build_dir jsonrpcpp) externals/jsonrpcpp
    cp -r $(get_build_dir popl) externals/popl
    cp externals/jsonrpcpp/lib/json.hpp externals/json.hpp
}
```

Reply:
Is that 'directory' indentatition systematic in LibreELEC?

Reply:
Yes, it's fairly common, to the point of almost being a "standard".